docs/source/01-01-talk-quickstart.md

```diff
@@ -178,32 +175,35 @@ You can now start the application by running:
 yarn watch:server
 ```
 
-Continue onto the [Running](#running) section for details on how to complete the
+Continue onto the [Setup](#setup) section for details on how to complete the
 installation and get started using Talk.
 
-## Running
+## Setup
+### Create Admin Account
 
-You can now navigate to
+With Talk running, you can now navigate to
 [http://127.0.0.1:3000/admin/install](http://127.0.0.1:3000/admin/install)
-and go through the admin installation. There you will be prompted to create your
-first admin account, and specify the domain whitelist for domains that are
-allowed to have the comment box on.
+and walk through the initial setup steps. 
+
+* First, enter your **Organization Name** and **Organization Contact Email**. This will appear in emails when inviting new team members.
+* Next, create your Admin user. You can specify an **Email Address**, **Username**, and **Password**
+* Finally, enter your list of **Permitted Domains**, read [here](talk/configuring-talk/#permitted-domains) about whitelisting domains 
```

Comment:
It's important to include a leading slash to ensure links are not broken.

```diff
- [here](talk/configuring-talk/#permitted-domains) about whitelisting domains
+ [here](/talk/configuring-talk/#permitted-domains) about whitelisting domains 
```

Reply:
gotcha, fixed